backend/webapi/Features/Parties/ProfileStatus.Model.cs

```diff
@@ -294,7 +294,7 @@ protected override StatusCode Compute(ProfileData profile)
             {
                 return profile switch
                 {
-                    { UserIsBCProvider: false } or { HasPrpAuthorizedLicence: false } => StatusCode.Locked,
+                    { UserIsBCProvider: false } => StatusCode.Locked,
```

Comment:
@james-hollinger Made the change to HasBCProviderCredential in the another PR related to Expanding PRP enrollment to pharmacists.
[prlink](https://github.com/bcgov/moh-pidp/pull/629#pullrequestreview-2450098360)